playbooks/common/openshift-cluster/redeploy-certificates/registry.yml

```diff
@@ -73,6 +73,9 @@
         --hostnames="{{ docker_registry_service_ip.results.clusterip }},docker-registry.default.svc.cluster.local,{{ docker_registry_route_hostname }}"
         --cert={{ openshift.common.config_base }}/master/registry.crt
         --key={{ openshift.common.config_base }}/master/registry.key
+        {% if openshift.common.version_gte_3_5_or_1_5 | bool %}
+        --expire-days={{ openshift_registry_cert_expire_days | default(730) }}
```

Comment:
Below, in some vars file I'm seeing defaults for expire days being set.

```diff
roles/openshift_ca/vars/main.yml
+openshift_ca_cert_expire_days: 1825
+openshift_master_cert_expire_days: 730

roles/openshift_master_certificates/vars/main.yml
+openshift_master_cert_expire_days: 730

roles/openshift_node_certificates/vars/main.yml
+openshift_node_cert_expire_days: 730
```

But here you are using `default(730)` in case `openshift_registry_cert_expire_days` is not defined. Should that not also be added to a roles `vars/`/`defaults/` file?